Pipfile

```diff
@@ -9,6 +9,7 @@ pillow = "*"
 pytest = "*"
 django-phonenumber-field = {extras = ["phonenumbers"], version = "*"}
 phonenumbers = "*"
+django-environ = "*"
```

Comment:
Can you explain the purpose of this package? 😄

Reply:
The "environ" library is to get environmental variables from a ".env" file.
Our usage is with the email arguments that need to be private, so I have opted to use a ".env" file instead of writing them inside the settings file.

Reply:
Ok thanks 👍